### ISSUE TITLE
Missing street segments in Seattle

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#3270]
Using @misaugstad's amazing new RouteBuilder, I noticed that we have some missing street segments. Here's an example from my neighborhood. I know these streets all have sidewalks (as I live here)

The gray streets are where we **have** data and the white streets are where we don't:

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/d2a9e9a5-3537-45a2-bdd5-6a7e8071a9f0)

@misaugstad, can we diagnose why we don't have those street segments in our database. Also, can we add them in now. We'd have to sweep through all Seattle neighborhoods and similarly find missing streets.

Also CC'ing @crescendochu.

**Comment by misaugstad** [comment]
Duplicate of #1814!

Some streets have been misclassified in OSM. We wrote up a process for adding some of those streets back in here: https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Adding-new-road-geometries-to-a-database

**Comment by jonfroehlich** [comment]
This is worth revisiting on a process level. When we deploy into a new city, we need to make sure that our standard practice is ensuring full road coverage (to the best we can). There are lots of examples of serious missing areas in Seattle:

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/80385cfb-8c84-4fed-bf42-150da59868dc)

![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/812b5be5-742c-4289-8572-a6fe253e7fe7)

**Comment by misaugstad** [comment]
Agreed, though I'm not sure how best to do this in an efficient way. We have [this wiki page](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Adding-new-road-geometries-to-a-database) which outlines how we can manually add new streets to our road network. But if 15% of the streets in Seattle are missing, that isn't going to cut it.

I did take a look at the South Lake Union area that you screenshotted there. It looks like 1 year ago, someone went through and fixed the roads to change them from "unclassified" to "residential".

We could try to create a semi-automated process, where we essentially go through some of the process of importing all of the data into our database, but we filter out streets that align with the streets that we already have in the database. There are bound to be plenty of outliers that we need to manually fix after this, but hopefully there will be far fewer outliers than missing streets..?

The above would still miss streets if they remain incorrectly listed as "unclassified". Just some initial thoughts for now!

**Comment by jonfroehlich** [comment]
Right... and I think part of this is just identifying that we are missing streets in the first place. The neighborhood in the top of the thread is where I live and I was surprised to see how much is missing--this is all residential neighborhood that should be audited.

But then, yes, once we identify where/how much missing streets we have, we need a way to fix...

**Comment by misaugstad** [comment]
Okay I forgot that I had actually thought about a lot of this stuff, and I beefed up the script that checks for missing GSV imagery enough that I felt like we could include the 'unclassified' streets in our databases. I've been doing this for the new databases, but I haven't gone back and added the data to all our old databases. Sounds like that's a step that I should bump up on the to-do list!

Closing as duplicate of #2547 where I actually talk about all of this.
